plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py

```diff
@@ -972,7 +975,11 @@ def _try_execute_wasm_query(
             json.dumps(query_msg),
         ]
 
-        return json.loads(self._execute_shell_command(command))
+        cli_stdout = self._execute_shell_command(command)
```

Comment:
just create a helper method for this pattern?

```
execute
try:
 load_json
 ...
 ```

Reply:
Yeah sure, today I care about speed though :D